apps/emqx_rule_engine/rebar.config

```diff
@@ -1,6 +1,7 @@
 %% -*- mode: erlang -*-
 
-{deps, [ {emqx, {path, "../emqx"}}
+{deps, [ {emqx, {path, "../emqx"}},
+         {jq, {git, "https://github.com/emqx/jq", {ref, "483cf9d8786efb6ab51defb42193f7f2406298f1"}}}
```

Comment:
I think that using commit hashes as references might confuse Rebar (and/or Mix) while building the release.  Can't remember exactly what the error is, but I believe we would need to use tags for the dependencies.

Reply:
Thanks, this git reference points to a buggy version so it would need to change before we merge this. I will make sure to create a tag and use that for the dependency before it is merged.

Reply:
I see. I will create a tag and change it to a tag but I think a hash should be more safe since one cannot change the files a hash refers to but this is not the case for tags.

Reply:
Fixed in [a785b06](https://github.com/emqx/emqx/pull/7384/commits/a785b063c8a798cfba9b21c4d12438e9ddfa95b6)